### PR TITLE
Test for both legacy and new events in bridges

### PR DESCRIPTION
**Pull request by maurelian**
Updates the L1 and L2 Standard bridge tests to ensure that both Legacy and Bedrock bridge events are emitted together. 

~~Tests are currently breaking.~~

OK, I was wrong, all these tests just needed some fixing up. Still they are good to have.

### CODE REVIEW

**Comment by maurelian** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #4744** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/4744" target="_blank">Graphite</a> 
    * **PR #4746** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/4746" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/optimism/4746?utm_source=stack-comment).

File: packages/contracts-bedrock/contracts/test/L1StandardBridge.t.sol [hunk 1]
```diff
@@ -78,6 +78,9 @@ contract L1StandardBridge_DepositETH_Test is Bridge_Initializer {
     function test_depositETH_succeeds() external {
         assertEq(address(op).balance, 0);
 
+        vm.expectEmit(true, true, true, true);
+        emit ETHDepositInitiated(alice, alice, 100, hex"");
```
Comment (tynes) [inline packages/contracts-bedrock/contracts/test/L1StandardBridge.t.sol:82]: should this be `hex"ff"`?

Comment (maurelian) [reply packages/contracts-bedrock/contracts/test/L1StandardBridge.t.sol:82]: oh probably.

**Comment by tynes** [comment]
@maurelian Looks like this needs a rebase

File: .circleci/config.yml [hunk 2]
```diff
@@ -238,7 +238,7 @@ jobs:
       - run:
           name: test and generate coverage
           command: yarn coverage:lcov
-          no_output_timeout: 15m
```
Comment (maurelian) [inline .circleci/config.yml:241]: This timed out on me: https://app.circleci.com/pipelines/github/ethereum-optimism/optimism/13108/workflows/164f734f-d1ae-4b40-b473-a624701f9a13/jobs/416089?invite=true#step-107-33

File: packages/contracts-bedrock/contracts/test/L2StandardBridge.t.sol [hunk 3]
```diff
@@ -16,10 +17,6 @@ import { OptimismMintableERC20 } from "../universal/OptimismMintableERC20.sol";
 contract L2StandardBridge_Test is Bridge_Initializer {
     using stdStorage for StdStorage;
 
-    function setUp() public override {
```
Comment (smartcontracts) [inline packages/contracts-bedrock/contracts/test/L2StandardBridge.t.sol:19]: Why is this removed?

Comment (maurelian) [reply packages/contracts-bedrock/contracts/test/L2StandardBridge.t.sol:19]: The super function is already public, so there's no need to override it with a Function That just calls it again

https://github.com/ethereum-optimism/optimism/blob/9f08dcbb93e5b2fba6ac76072048b735f429cbd9/packages/contracts-bedrock/contracts/test/CommonTest.t.sol

File: packages/contracts-bedrock/contracts/test/L1StandardBridge.t.sol [hunk 4]
```diff
@@ -338,7 +338,10 @@ contract L1StandardBridge_DepositERC20_Test is Bridge_Initializer {
             innerMessage
         );
 
-        // ERC20 Deposit Initiated event emitted by the StandardBridge Contract
+        // Should emit both the bedrock and legacy events
+        vm.expectEmit(true, true, true, true);
```
Comment (tynes) [inline packages/contracts-bedrock/contracts/test/L1StandardBridge.t.sol:342]: It looks like there is an overload of `expectEmit` that also accepts the `address` as the final arg? Do you mind updating these to ensure the correct contracts are emitting the events?

Comment (maurelian) [reply packages/contracts-bedrock/contracts/test/L1StandardBridge.t.sol]: OK, done.

**Review by tynes — APPROVED** [review]

**Review by zhwrd — APPROVED** [review]
